### PR TITLE
fix: create box from the client

### PR DESCRIPTION
**Pull request by JoeSzymkowiczFiveM**
## Description

This addresses issue 4, now spawning the electrical box when the player is near the jewelry store. Also makes the rayfire thread only active inside that same zone.

## Checklist

- [] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [X] My code fits the style guidelines.
- [X] My PR fits the contribution guidelines.

### CODE REVIEW

File: client.lua [hunk 1]
```diff
@@ -256,20 +274,36 @@ RegisterNetEvent('qb-jewelery:client:alarm', function()
...
-                    local RayFire = GetRayfireMapObject(Config.Cabinets[i].coords.x, Config.Cabinets[i].coords.y, Config.Cabinets[i].coords.z, 1.4, Config.Cabinets[i].rayFire)
-                    SetStateOfRayfireMapObject(RayFire, 9)
+lib.zones.sphere({
+    coords = vec3(Config.Cabinets[1].coords.x, Config.Cabinets[1].coords.y, Config.Cabinets[1].coords.z),
+    radius = 80,
+    --debug = true,
+    onEnter = function()
+        createElectricalBox()
+        CreateThread(function()
+
```
Comment (D4isDAVID) [inline client.lua:284]:
```suggestion
            insideJewerly = true
```
Seems to me like you forgot this

File: client.lua [hunk 2]
```diff
@@ -49,8 +54,21 @@ if Config.UseTarget then
...
         }
-    })
-else
+        exports.ox_target:addLocalEntity(electricalBoxEntity, options)
+    end
+end
+
+local function removeElectricalBox()
+    if Config.UseTarget then
+        exports.ox_target:removeLocalEntity({electricalBoxEntity}, 'qb-jewelery:electricalBox')
```
Comment (D4isDAVID) [inline client.lua:63]:
```suggestion
        exports.ox_target:removeLocalEntity(electricalBoxEntity, 'qb-jewelery:electricalBox')
```
A single entity is removed here

**Review by D4isDAVID — CHANGES_REQUESTED** [review]

File: client.lua (hunk 1 above)
Comment (JoeSzymkowiczFiveM) [reply client.lua:285]: Good catch. Corrected.

File: client.lua (hunk 2 above)
Comment (JoeSzymkowiczFiveM) [reply client.lua]: Amended syntax for removeLocalEntity entity.

File: client.lua [hunk 3]
```diff
@@ -29,16 +31,19 @@ local function DrawText3D(coords, text)
...
-        size = vec3(1, 1, 2.4),
-        rotation = Config.Electrical.w,
-        debug = true,
-        options = {
+local function createElectricalBox()
+    electricalBoxEntity = CreateObject(`tr_prop_tr_elecbox_01a`, Config.Electrical.x, Config.Electrical.y, Config.Electrical.z, false, false, false)
+    while not DoesEntityExist(electricalBoxEntity) do
+        Wait(0)
+    end
+    Wait(100)
```
Comment (D4isDAVID) [inline client.lua:39]:
```suggestion
```
This is unnecessary

**Review by D4isDAVID — CHANGES_REQUESTED** [review]

File: client.lua (hunk 3 above)
Comment (JoeSzymkowiczFiveM) [reply client.lua]: Removed this.

**Review by D4isDAVID — APPROVED** [review]
LGTM + Tested and works

**Review by YvanCywan — APPROVED** [review]

**Pull request merged**